src/OrchardCore.Modules/OrchardCore.Lists/Drivers/ListPartDisplayDriver.cs

```diff
@@ -134,6 +135,17 @@ private ShapeResult InitializeDisplayListPartDetailAdminShape(ListPart listPart,
         }))
             .Location("DetailAdmin", "Content:10");
     }
+    private ShapeResult InitializeDisplayListPartDetailAdminSearchPanelShape()
+    {
+        return Initialize<ListPartViewModel>("ListPartDetailAdminSearchPanel", async model =>
+            {
+                var listPartFilterViewModel = new ListPartFilterViewModel();
+                await _updateModelAccessor.ModelUpdater.TryUpdateModelAsync(listPartFilterViewModel, Prefix);
+                model.ListPartFilterViewModel = listPartFilterViewModel;
+
+             })
+            .Location("DetailAdmin", "Content:5");
+    }
```

Comment:
```suggestion
    private ShapeResult InitializeDisplayListPartDetailAdminSearchPanelShape() =>
        Initialize<ListPartViewModel>("ListPartDetailAdminSearchPanel", async model =>
        {
             var listPartFilterViewModel = new ListPartFilterViewModel();
             await _updateModelAccessor.ModelUpdater.TryUpdateModelAsync(listPartFilterViewModel, Prefix);
             
             model.ListPartFilterViewModel = listPartFilterViewModel;

        })
        .Location("DetailAdmin", "Content:5");
```

Reply:
Don't do this please. Bodies expressions are for one-line code only. We had a conversation about it during one of the meetings

Reply:
Revert it back